src/main/java/thewaypointers/trafficsimulator/simulation/factories/xml/models/RoadXML.java

```diff
@@ -0,0 +1,63 @@
+package thewaypointers.trafficsimulator.simulation.factories.xml.models;
+
+import thewaypointers.trafficsimulator.common.Direction;
+
+public class RoadXML {
+    private String origin;
+    private String destination;
+    private String length;
+    private String direction;
+
+    public String getOrigin() {
+        return origin;
+    }
+
+    public void setOrigin(String origin) {
+        this.origin = origin;
+    }
+
+    public String getDestination() {
+        return destination;
+    }
+
+    public void setDestination(String destination) {
+        this.destination = destination;
+    }
+
+    public long getLength() {
+        String lengTolong = length.trim();
+
+        try {
+            long lenghtInLong = Long.parseLong(lengTolong);
+            return lenghtInLong;
+        } catch (Exception ex) {
+            System.out.println("Error with casting length to long");
+            System.out.println(ex.getMessage());
+        }
+        return 0;
+    }
+
+    public void setLength(String length) {
+        this.length = length;
+    }
+
+    public Direction getDirection() {
```

Comment:
you can't reference the direction variable then because it would be in static context in a non static class